go/libraries/doltcore/remotestorage/chunk_store.go

```diff
@@ -101,6 +103,8 @@ func NewDoltChunkStore(ctx context.Context, nbf *types.NomsBinFormat, org, repoN
 		csClient = RetryingChunkStoreServiceClient{csClient}
 	}
 
+	evt := events.NewEvent(eventsapi.ClientEventType_REMOTEAPI_GET_REPO_METADATA)
```

Comment:
Rather than putting:

    events.GlobalCollector.CloseEventAndAdd(evt)

in multiple places, it would be better to:

    defer events.GlobalCollector.CloseEventAndAdd(evt)

Reply:
The same thing for each of the calls below.